RecoTracker/Configuration/python/customizePixelOnlyForProfiling.py

```diff
@@ -23,12 +25,13 @@ def customizePixelOnlyForProfilingGPUOnly(process):
 #
 # The same customisation can be also used on the SoA CPU workflow, running up to the
 # tracks and vertices on the CPU in SoA format, without conversion to legacy format.
+# TODO: does exist a module that forces the copy to host for Alpaka?
```

Comment:
I think the change here in `customizePixelOnlyForProfilingGPUWithHostCopy` does exactly that: it _consumes_ both device and host version of the `pixelTracksAlpaka` and `pixelVerticesAlpaka` collections.